app/desmos/cmd/chainlink/types/getter.go

```diff
@@ -13,8 +13,13 @@ import (
 
 // ChainLinkReferenceGetter allows to get all the data needed to generate a ChainLinkJSON instance
 type ChainLinkReferenceGetter interface {
-	// GetMnemonic returns the mnemonic
-	GetMnemonic() (string, error)
+
+	// GetIsSingleSignatureAccount returns if the target account is single signature account
+	GetIsSingleSignatureAccount() (bool, error)
```

Comment:
```suggestion
	IsSingleSignatureAccount() (bool, error)
```
I would rename this like this to make it clearer